website/blog/2023-02-07-5-database-alternatives-mongodb-2023.md

```diff
@@ -0,0 +1,137 @@
+
+MaxScale NoSQL protocol module is a communication protocol between MariaDB and MongoDB, which enables the use of a MariaDB server or cluster as the backend for an application that uses a MongoDB client library.
+
+Users can interact with MariaDB databases using NoSQL-style commands, which are then converted into SQL and executed on the MariaDB backend.
+The responses from MariaDB are then transformed into the format required by the MongoDB client library and application.
+
+The MariaDB NoSQL protocol module is suitable for users that require relational database support, scalability, and integration with existing MariaDB infrastructure.
+Note that MaxScale is published under a proprietary licence (BSL), meaning you're still faced with similar concerns of vendor lock-in.
+
+### 5. GaussDB(for Mongo)
+
+[GaussDB(for Mongo)](https://www.huaweicloud.com/intl/en-us/product/gaussdbformongo.html) is a proprietary cloud-native database that's compatible with MongoDB.
+Similar to DocumentDB, it offers decoupled storage and compute, enabling seamless and flexible scaling.
+
+GaussDB is especially good for gaming applications that require enterprise-class performance, high flexibility and storage capacity, and a friendly UI.
+The ease of adding compute nodes makes it perfect for online gaming scenarios with high-concurrency levels.
+
+### Notable Mention: ToroDB
+
+ToroDB is not a part of this list due to the limited activity on the project in recent years; however, it's another MongoDB alternative that's worthy of mention.
+
+[ToroDB](https://www.torodb.com/) is an open source database service built to read and convert MongoDB NoSQL to SQL.
+With PostgreSQL as the storage layer, ToroDB doesn't use jsonb; Instead, it uses a relational approach to store data.
+ToroDB also implements the MongoDB protocol, making it compatible with most MongoDB applications.
+
+Despite ToroDB's potential as a viable MongoDB alternative, one big red flag is the absence of consistent updates or improvements on the database project for a while now, making it an unlikely choice for would-be users.
+In fact, there hasn’t been any activity on the project since 2019.
+Nevertheless, the project is still available on GitHub, and users can happily experiment with it.
+
+## Choosing the MongoDB alternative for your app
+
+Before choosing an alternative to MongoDB, it's important to take several factors into consideration, including ease of use, flexibility, vendor independence, scalability, cost, and maintenance.
+
+For users with high availability and storage demands, little concern for vendor lock-in, and the budget to accommodate it, choosing a proprietary solution between DocumentDB, CosmosDB, or GaussDB would be the best option.
+
+For developers or organizations that would love to be part of a growing community of open source enthusiasts with the freedom to run, manage, and host their database as they wish, FerretDB would be an ideal choice.
```

Comment:
@ptrfarkas Here, I summarize the lists and mention FerretDB as the open source option.